validation/run_single_q0.py

```diff
@@ -0,0 +1,36 @@
+if __name__ == "__main__":
```

Comment:
Since `validation/run_single_q0.py` never get run in the CI can we get this cleaned up a bit with some comments on how this is expected to be used in validation?

Reply:
I'd say this falls under #1241 for now -- since we don't have `validation/run_single.py` tested either -- since this requires ROOT.

Reply:
Okay good enough